.github/workflows/release-e2e.yaml

```diff
@@ -0,0 +1,51 @@
+name: release-e2e
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+
+jobs:
+  release-e2e-kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '~1.18'
+
+    - name: install_kind_binary
+      env:
+        KIND_VERSION: v0.12.0
+        KIND_OS: kind-linux-amd64
+      run: |
+        curl -sLo kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/${KIND_OS}"
+        chmod +x kind
+        sudo mv kind /bin/
+
+    - name: start_kind_cluster
+      run: |
+        # /bin/kind create cluster
+        make kind-cluster
+
+
+    - name: install_kubectl
+      run: |
+        curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+        sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+    - name: release-install
+      needs: [start_kind_cluster, install_kubectl]
+      run: |
+        kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+        kubectl wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook --timeout=60s
+        kubectl apply -f https://github.com/operator-framework/rukpak/releases/download/v0.3.0/rukpak.yaml
+
+    - name: load_bundles
+      needs: [release-install]
+      run: |
+        make kind-load-bundles
+
+    - name: Run e2e tests
+      needs: [load_bundles]
+      run: make e2e KIND=/bin/kind
```

Comment:
`make e2e` actually builds the container and stands up a new cluster, which is redundant... We can run the e2e tests via ginkgo directly? Something like
```
go run github.com/onsi/ginkgo/v2/ginkgo ./test/e2e
```
we could potentially add a new target for this as well.

Reply:
We already have an existing `test-e2e` Makefile target which runs ginkgo under-the-hood. I'd recommend re-using that target, and refactoring this check to avoid using the `e2e` target like Dan had mentioned as that target is performing a bunch of installation check that aren't relevant to running e2e tests against an existing rukpak installation that uses the release tagged images.

Reply:
Switched targets.